src/parser/jobs/war/Gauge.js

```diff
@@ -58,47 +58,43 @@ export default class Gauge extends Module {
 
 	constructor(...args) {
 		super(...args)
-		this.addHook('cast', {by: 'player'}, this._onCast)
 		this.addHook('death', {to: 'player'}, this._onDeath)
+		this.addHook('cast', {by: 'player', abilityId: Object.keys(RAGE_SPENDERS).map(Number)}, this._onSpenderCast)
+		this.addHook('cast', {by: 'player', abilityId: Object.keys(RAGE_GENERATORS).map(Number)}, this._onBuilderCast)
```

Comment:
You should be using the `'combo'` hook for the builders instead of `'cast'`. It should just work with no other changes, the combo events are fabbed as duplicates of the cast events.

Reply:
Was an oversight from me. Was testing something, and forgot to change it back to combos.